<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>export_setting_dialog</class>
 <widget class="QDialog" name="export_setting_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>203</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QRadioButton" name="exportSingle">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>87</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>导出第1项</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="exportSelected">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>87</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>导出选中项</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="exportAll">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>87</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>导出全部</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="remember_cheBox2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>151</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>记住我的选择，不再提醒</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OkButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>确定(O)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelButton">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>160</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>取消(C)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ApplyButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>160</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>应用(A)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
